addons/overpressure/config.cpp

```diff
@@ -8,7 +8,7 @@ class CfgPatches {
         requiredVersion = REQUIRED_VERSION;
         requiredAddons[] = {"ace_common"};
         author = ECSTRING(common,ACETeam);
-        authors[] = {"commy2","KoffeinFlummi","esteldunedain"};
+        authors[] = {"commy2","KoffeinFlummi","esteldunedain", "bluefield"};
```

Comment:
IMO this is reserved for people who have contributed significantly in either the creation or the refactoring of a component, https://github.com/acemod/ACE3/blob/master/AUTHORS.txt would be more appropriate for this PR.